src/main/java/usecase/alter_entry/AlterEntryInputPort.java

```diff
@@ -0,0 +1,7 @@
+package usecase.alter_entry;
+
+import billgates.database.QueryEntryData;
+
+public interface AlterEntryInputPort {
+    void alterEntry(int billId, int entry_id, Object new_value,String alter_column);
```

Comment:
In this case, there are many parameters. 

It is better to add a `AlterEntryRequestModel`.